R-package/src/install.libs.R

```diff
@@ -56,6 +56,7 @@ if (!use_precompile) {
       cmake_cmd <- paste0(cmake_base, " -DCMAKE_GENERATOR_PLATFORM=x64 ")
       tryVS <- system(paste0(cmake_cmd, " .."))
       if (tryVS == 1) {
+        system("rm -rf .")
```

Comment:
It works with your method.

`rm rf` works, but I was denied access to remove `.` or `..` apparently.